core/templates/dev/head/base.html

```diff
@@ -225,9 +225,35 @@
                             </ul>
                           </li>
                         {% endif %}
-                      </ul>
-                    </div>
-                  {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if="windowWidth > 700"></create-exploration-button>
+                          <li ng-if="windowWidth > 700" class="dropdown oppia-clickable-dropdown pull-right" >
```

Comment:
We repeat the ng-if condition here many times. Can we do it just once for the entire branch?